src/components/Modal.js

```diff
@@ -0,0 +1,119 @@
+              </p>
+              <p className="modal-icon-text">
+                <FontAwesomeIcon icon={faUserGroup} /> 4 guests
+              </p>
+            </div>
+            <div className="modal-text-row d-flex">
+              <span className="modal-icon-text">
+                <FontAwesomeIcon icon={faShower} />{" "}
+                {selectedApartment.bathrooms} bathrooms{" "}
+              </span>
+              <span className="modal-icon-text">
+                <FontAwesomeIcon icon={faRulerCombined} />{" "}
+                {selectedApartment.square_footage} sq/ft
+              </span>
+            </div>
+            <div style={{ marginTop: "6vh" }}>
+              <div style={{ fontSize: "3vh", fontWeight: "600" }}>
+                Popular Amenities
+              </div>
+              imperdiet, nulla et dictum interdum, nisi lorem egestas vitae scel
+              {showMore ? (
+                <span>
+                  erisque enim ligula venenatis dolor. Maecenas nisl est,
+                  ultrices nec congue eget, auctor vitae massa. Fusce luctus
+                  vestibulum augue ut aliquet. Nunc sagittis dictum nisi, sed
+                  ullamcorper ipsum dignissim ac. In at libero sed nunc
+                  venenatis imperdiet sed ornare turpis. Donec vitae dui eget
+                  tellus gravida venenatis. Integer fringilla congue eros non
+                  fermentum. Sed dapibus pulvinar nibh tempor porta. erisque
+                  enim ligula venenatis dolor. Maecenas nisl est, ultrices nec
+                  congue eget, auctor vitae massa. Fusce luctus vestibulum augue
+                  ut aliquet. Nunc sagittis dictum nisi, sed ullamcorper ipsum
+                  dignissim ac. In at libero sed nunc venenatis imperdiet sed
+                  ornare turpis
+                </span>
+              ) : (
+                <span id="dots">...</span>
+              )}
+            </div>
+            <p onClick={toggleShowMore} className="subtle-btn">
+              {showMore ? (
+                <>
+                  <FontAwesomeIcon icon={faChevronLeft} /> Read less
+                </>
+              ) : (
+                <>
+                  Read more <FontAwesomeIcon icon={faChevronRight} />
+                </>
+              )}
+            </p>
+          </div>
+        </div>
+        <FontAwesomeIcon
+          className="modal-close-btn"
+          onClick={() => handleClick()}
+          icon={faTimesCircle}
+        />
+        <div className="modal-img-cont d-flex">
+          <img className="modal-img" src={selectedApartment.image} alt="unit" />
+        </div>
+      </div>
+    </div>
+  )
+}
+
+// t.string "street"
+//     t.string "unit"
+//     t.string "city"
+//     t.string "state"
+//     t.integer "square_footage"
+//     t.string "price"
+//     t.integer "bedrooms"
+//     t.float "bathrooms"
+//     t.string "pets"
+//     t.text "image"
```

Comment:
I totally understand why this is here, but I would remove it before merging to main.